bin/MT_LoadCheck.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Aug 29 10:37:32 2016
+
+@author: kun
+"""
+
+import robot as rob
+import time
+
+CMD_TIMEOUT = 500000
+
+
+try:
+
+    cmds=['Initialize','Cooldown']
+    for scmd in cmds:
+        tskStat, sampleStat, exception = rob.runATask(scmd, CMD_TIMEOUT)
+        if (tskStat.lower()!="done"):
+            raise Exception(scmd+" "+tskStat+" with exception "+exception)
```

Comment:
I think these are caught right now by the try statement - add an except clause sending them to errout?